### DISCUSSION TITLE
How to connect to nx cloud?

### DISCUSSION THREAD

**Discussion by ZebraFlesh** [nrwl/nx#22950] [General]
I've inherited an nx project hosted on gitlab.com. When I run `nx connect`, I get the response "This workspace already has Nx Cloud set up". My nx.json file does not have an `nxCloudAccessToken` property. When I get a failed build on my local machine, visiting the provided link for logs gives me a page that says "Contact the workspace owner to see this run."

How can I resolve this? (The Support "link" on the nx.app website is currently erroring out.)

**Comment by jaysoo** [comment]
@ZebraFlesh It sounds like someone has connected the repo to Nx Cloud already. If the support link is not working for you, you can email `cloud-support@nrwl.io` with your request.
